site/src/api/typesGenerated.ts

```diff
@@ -337,10 +338,13 @@ export interface Entitlements {
   readonly warnings: string[]
   readonly errors: string[]
   readonly has_license: boolean
-  readonly experimental: boolean
   readonly trial: boolean
+  readonly experimental: boolean
 }
 
+// From codersdk/experiments.go
+export type Experiments = Experiment[]
```

Comment:
Someone familiar with ts should comment if this is ok. I think it is, but would like someone to confirm.

Reply:
I was confused with the name collision, this is good :+1: